kafka/librdkafka/bundle-windows.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# This should be run from the MSYS2 MinGW 64-bit shell to build librdkafka_windows.a
```

Comment:
Our Go client packaging workflow relies on all library assets already being built by librdkafka builders, so it would be best if we did the same thing here.
That means moving these build steps to a librdkafka Travis or AppVeyor worker, the artifacts will be automatically uploaded to the S3 bucket. Modify librdkafka/packaging/nuget/packaging.py to include the mingw build in its StaticPackage class, the resulting tar ball is what is used by the Go import.sh.

Reply:
Working on this here https://github.com/edenhill/librdkafka/pull/3130

Reply:
@edenhill 
I believe that, if the changes in https://github.com/edenhill/librdkafka/pull/3130 are merged, only `bundle-import.sh` will need updated for the existing packaging workflow for confluent-kafka-go to support the Windows MinGW build. That said, I don't think _I'll_ be able to run `release.py`, as it requires AWS creds. How should I acquire `build_windows.go` and `librdkafka_windows.a` to add to this PR?